src/components/bestSellers/BestSellersData.js

```diff
@@ -0,0 +1,48 @@
+import { db } from '../../api/firebase'
+import { collection, getDocs } from 'firebase/firestore'
+import { query, orderBy, limit } from "firebase/firestore";
+import { useEffect, useState } from 'react'
+import { BookCard } from '../bookListElement/BookCard';
+import { StyledBestSellersData, StyledH2 } from '../bestSellers/BestSellers.styled'
+import { TextAndButtons } from './BestSellers.styled'
+
+
+export const BestSellersData = () => {
+    const [books, setBooks] = useState([])
```

Comment:
zmieniłabym nazwę na [bestsellers, setBestsellers] , bo to nie są wszystkie książki